src/main/java/ch/admin/bar/siard2/jdbc/PostgresMetaColumns.java

```diff
@@ -355,8 +396,15 @@ private int getPrecision(int iPrecision, int iType, String sTypeName)
           iPrecision = iMAX_TEXT_LENGTH; 
           break;
         case BIT:
-        case VARBIT: 
-          iPrecision = (iPrecision + 7) / 8; 
+        case VARBIT:
+          // Get the actual bit length directly from the system catalog
+          String schema = this.getString(2);
+          String table = this.getString(3);
+          String column = this.getString(4);
```

Comment:
the triple of schema, table and column was already used before. Maybe this would be worth it's own object?